code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -5,26 +5,28 @@
 /datum/category_item/player_setup_item/general/equipment/load_character(var/savefile/S)
 	S["underwear"]	>> pref.underwear
 	S["undershirt"]	>> pref.undershirt
+	S["socks"]		>> pref.socks
 	S["backbag"]	>> pref.backbag
 	S["gear"]		>> pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/save_character(var/savefile/S)
 	S["underwear"]	<< pref.underwear
 	S["undershirt"]	<< pref.undershirt
+	S["socks"]		<< pref.socks
 	S["backbag"]	<< pref.backbag
 	S["gear"]		<< pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/gather_load_query()
-	return list("ss13_characters" = list("vars" = list("underwear", "undershirt", "backbag", "gear"), "args" = list("id")))
+	return list("ss13_characters" = list("vars" = list("underwear", "undershirt", "socks", "backbag", "gear"), "args" = list("id")))
 
 /datum/category_item/player_setup_item/general/equipment/gather_load_parameters()
 	return list(":id" = pref.current_character)
 
 /datum/category_item/player_setup_item/general/equipment/gather_save_query()
-	return list("ss13_characters" = list("underwear", "undershirt", "backbag", "gear", "id" = 1))
+	return list("ss13_characters" = list("underwear", "undershirt", "socks", "backbag", "gear", "id" = 1))
 
 /datum/category_item/player_setup_item/general/equipment/gather_save_parameters()
-	return list(":underwear" = pref.underwear, ":undershirt" = pref.undershirt, ":backbag" = pref.backbag, ":gear" = list2params(pref.gear), ":id" = pref.current_character)
+	return list(":underwear" = pref.underwear, ":undershirt" = pref.undershirt, ":socks" = pref.socks, ":backbag" = pref.backbag, ":gear" = list2params(pref.gear), ":id" = pref.current_character)

```

Comment:
Real quick, what format should socks be under? What's the maximum length, and do they contain letters?